packages/web/src/components/gcds-notice/test/gcds-notice.spec.tsx

```diff
@@ -0,0 +1,260 @@
+              GC Design System notice
+            </gcds-heading>
+            <slot></slot>
+          </div>
+        </section>
+      </mock:shadow-root>
+      <gcds-text>
+        Provide additonal information
+      </gcds-text>
+    </gcds-notice>
+    `);
+  });
+
+  it('renders - title tag h5', async () => {
+    const page = await newSpecPage({
+      components: [GcdsNotice],
+      html: `<gcds-notice notice-title-tag="h5" notice-title="GC Design System notice" type="warning">
+        <gcds-text>Provide additonal information</gcds-text>
+      </gcds-notice>`,
+    });
+    expect(page.root)
+      .toEqualHtml(`<gcds-notice notice-title-tag="h5" notice-title="GC Design System notice" type="warning">
+      <mock:shadow-root>
+        <section class="gcds-notice notice--type-warning">
+          <gcds-icon class="notice__icon" name="exclamation-triangle" size="h4"></gcds-icon>
+          <div>
+            <gcds-heading class="notice__heading" margintop="300" tag="h5">
+              GC Design System notice
+            </gcds-heading>
+            <slot></slot>
+          </div>
+        </section>
+      </mock:shadow-root>
+      <gcds-text>
+        Provide additonal information
+      </gcds-text>
+    </gcds-notice>
+    `);
+  });
+
+  it('does not render - no type', async () => {
+    const page = await newSpecPage({
+      components: [GcdsNotice],
+      html: `<gcds-notice notice-title-tag="h2" notice-title="GC Design System notice">
+        <gcds-text>Provide additonal information</gcds-text>
+      </gcds-notice>`,
+    });
+    expect(page.root)
+      .toEqualHtml(`<gcds-notice notice-title-tag="h2" notice-title="GC Design System notice">
+      <mock:shadow-root>
+      </mock:shadow-root>
+      <gcds-text>
+        Provide additonal information
+      </gcds-text>
+    </gcds-notice>
+    `);
+  });
+
+  it('does not render - no notice title', async () => {
+    const page = await newSpecPage({
+      components: [GcdsNotice],
+      html: `<gcds-notice notice-title-tag="h2" type="danger">
+        <gcds-text>Provide additonal information</gcds-text>
+      </gcds-notice>`,
+    });
+    expect(page.root).toEqualHtml(`<gcds-notice notice-title-tag="h2" type="danger">
+      <mock:shadow-root>
+      </mock:shadow-root>
+      <gcds-text>
+        Provide additonal information
+      </gcds-text>
+    </gcds-notice>
+    `);
+  });
+
+  it('does not render - no passed message', async () => {
+    const page = await newSpecPage({
+      components: [GcdsNotice],
+      html: `<gcds-notice notice-title-tag="h5" notice-title="GC Design System notice" type="danger">
+      </gcds-notice>`,
+    });
+    expect(page.root)
+      .toEqualHtml(`<gcds-notice notice-title-tag="h5" notice-title="GC Design System notice" type="danger">
+      <mock:shadow-root>
+      </mock:shadow-root>
+    </gcds-notice>
+    `);
+  });
+
+  it('does not render - no notice title tag', async () => {
```

Comment:
Can you add just one more test:


```
  it('does not render - invalid notice title tag', async () => {
```
where it has the tag set to an invalid value?